narwhal/config/src/lib.rs

```diff
@@ -139,6 +145,16 @@ pub struct Parameters {
     pub network_admin_server: NetworkAdminServerParameters,
 }
 
+impl Parameters {
+    fn default_header_num_of_batches_threshold() -> usize {
+        32
+    }
+
+    fn default_max_header_num_of_batches() -> usize {
+        1000
```

Comment:
Are we sure we want 1000 here rather than 100? This could lead to an unwieldy block which sync could create a straggler.

Nit: on every occurrence of `1000` in Rust, we can separate values using `1_000`.

Reply:
@huitseeker I've described the logic why I put `1000` here as part of this https://github.com/MystenLabs/sui-operations/pull/496 . More specifically:

> The max_header_num_of_batches has been set to 1000 - although this is something that needs to be reviewed. Assuming a rate of ~200ms for batch creation & dissemination, (so ~5 batches /sec) this will give us a headroom of including in the next header the batches created on the previous 200 sec = 3min 30 sec . In periods of long asynchrony this might make the system recover slowly but its a protection mechanism anyways - if we consider acceptable we can increase the size.

I am totally ok to go with a lower number but for now didn't want to be too restrictive